recipes/bioconductor-dexseq/pythonpath.patch

```diff
@@ -0,0 +1,16 @@
+--- inst/python_scripts/dexseq_count.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_count.py.new	2017-06-16 14:14:38.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
++
+ import sys, itertools, optparse, warnings
+ 
+ optParser = optparse.OptionParser( 
+--- inst/python_scripts/dexseq_prepare_annotation.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_prepare_annotation.py.new	2017-06-16 14:14:55.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
```

Comment:
I see it more from the other side, if people want to modify things on there own they hopefully know what they are doing and we prevent them from doing it.
I think the majority of packages are using /usr/bin/env and maybe we should make this best practice?